File: PeachPie Extension Libraries/Peachpie.Base.Ext/.sdk/Php/Threading/Tasks/ManagedTaskException.php
<?php

namespace Php\Threading\Tasks;

use Exception;

class ManagedTaskException extends Exception
{
    /**
     * @param string|null $message
     * @param int|null $inner
     * @throws ManagedTaskException
     */
    public function __construct(string $message = null, int $inner = null)
    {
        throw new ManagedTaskException($message, $inner);
    }
}

File: PeachPie Extension Libraries/Peachpie.Base.Ext/src/Threading/Tasks/ManagedTaskException.cs

using Pchp.Library.Spl;

namespace Php.Threading.Tasks;

public class ManagedTaskException : Exception
{
    public ManagedTaskException(string message)
        : base(message) { }

    public ManagedTaskException(string message, long inner)
        : base(message, inner) { }
}